frontend/src/component/common/Search/SearchSuggestions/SearchInstructions/SearchInstructions.tsx

```diff
@@ -57,6 +61,10 @@ export const SearchInstructions: VFC<ISearchInstructionsProps> = ({
                         condition={filter.options.length > 0}
                         show={
                             <StyledCode
+                                tabIndex={0}
```

Comment:
previously we only supported clicks, now we support on  enter when tab index is selected